nginx/domains/status.ffmuc.net.conf

```diff
@@ -0,0 +1,34 @@
+upstream uptime_kuma_backend {
+   server docker07.ov.ffmuc.net:8083;
+   keepalive 32;
+}
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    listen 80;
+    listen [::]:80;
+
+    server_name status.ffmuc.net;
+
+    client_max_body_size 0;
+
+    location / {
+        resolver 5.1.66.255 valid=30s;
```

Comment:
Is this really still needed ? And can there be multiple resolvers? As this creates a SPOF on vienna. please also add 185.150.99.255

Reply:
Not needed. Yet this SPOF is still present over at the unifi conf